lib/utils/jsonPathUtils.ts

```diff
@@ -58,4 +59,48 @@ export class JsonPathUtils {
     }
     return result;
   }
+
+  public static changePropertyNameRecursively(
+    obj: IInternalPresentationDefinition,
+    currentPropertyName: string,
+    newPropertyName: string
+  ): any {
+    console.log(newPropertyName);
```

Comment:
Either use the debug package to log and be able to define when to log, or remove the console.log

Reply:
removed altogether